test/README.md

```diff
@@ -53,8 +53,27 @@ AMQP URL
 
     export RACKHD_AMQP_URL=[amqp://<url>:<port> default=amqp://localhost:9091]
 
-Specify test groups
+OS Installer locations
+
+    Specify specific OS repository path:
+
+    export RACKHD_CENTOS_REPO_PATH=[location to http mirror for all CentOS images]
+    export RACKHD_ESXI_REPO_PATH=[location to http mirror for all ESXi images]
+    export RACKHD_UBUNTU_REPO_PATH=[location to http mirror for all Ubuntu images]
+    export RACKHD_SUSE_REPO_PATH=[location to http mirror for all SUSE images]
```

Comment:
There will be more and more such configuration entries as RackHD tests grow up, a single file to manage all these configuration will be more reasonable to me. 
I'm OK to move the test config file as a standalone story to track.